.github/workflows/build-check.yml

```
@@ -0,0 +1,37 @@
+        uses: oven-sh/setup-bun@v2
+
+      - name: Cache Dependencies
+        uses: actions/cache@v4
+        with:
+          path: ~/.bun/install/cache
+          key: ${{ runner.os }}-bun-deps-${{ hashFiles('bun.lockb') }}
+          restore-keys: |
+            ${{ runner.os }}-bun-deps-
+
+      - name: Install Dependencies
+          run: bun install
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 34: mapping values are not allowed in this context

(syntax-check)